dart/lib/src/protocol/breadcrumb.dart

```diff
@@ -87,25 +87,25 @@ class Breadcrumb {
     if (category != null) {
       json['category'] = category;
     }
-    if (data != null && data.isNotEmpty) {
+    if (data?.isNotEmpty ?? false) {
       json['data'] = data;
     }
-    if (level != null) {
-      json['level'] = level.name;
-    }
+
+    json['level'] = level.name;
```

Comment:
why so? https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/ everything is optional

Reply:
Because previously level had a default value, that's why the migration tool made it non-nullable.
I guess it is also a little bit nicer to work with because you don't have to check if level is null in https://pub.dev/documentation/sentry/latest/sentry/SentryOptions/beforeBreadcrumb.html 

I could imagine that dropping Breadcrumbs because of the level is quite common.

However I'm fine with either way.

Reply:
`timestamp` is never null, because the ctor sets it automatically, so I'd prefer to keep the same behavior, we could make it nullable (level), but setting a default value `info`, its like this across other SDKs too.
because of the `timestamp`, `json.isNotEmpty` would always return false anyway